cmd/juju/caas/add.go

```diff
@@ -160,9 +161,12 @@ type AddCAASCommand struct {
 func NewAddCAASCommand(cloudMetadataStore CloudMetadataStore) cmd.Command {
 	store := jujuclient.NewFileClientStore()
 	cmd := &AddCAASCommand{
-		OptionalControllerCommand: modelcmd.OptionalControllerCommand{Store: store},
-		cloudMetadataStore:        cloudMetadataStore,
-		store:                     store,
+		OptionalControllerCommand: modelcmd.OptionalControllerCommand{
+			Store:       store,
+			EnabledFlag: feature.MultiCloud,
```

Comment:
Is this part of the add-k8s command? If so, I don't think we need the multicloud feature flag on this right?